packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1983,6 +1993,129 @@ describe('TransactionController', () => {
     });
   });
 
+  describe('updateTransactionSendFlowHistory', () => {
+    it('appends sendFlowHistory entries to transaction meta', async () => {
+      const controller = newController();
+      const mockSendFlowHistory = [
+        {
+          entry:
+            'sendFlow - user selected transfer to my accounts on recipient screen',
+          timestamp: 1650663928211,
+        },
+      ];
+      await controller.addTransaction({
+        from: ACCOUNT_MOCK,
+        to: ACCOUNT_MOCK,
+      });
+      const addedTxId = controller.state.transactions[0].id;
+      controller.updateTransactionSendFlowHistory(
+        addedTxId,
+        0,
+        mockSendFlowHistory,
+      );
+
+      expect(controller.state.transactions[0].sendFlowHistory).toStrictEqual(
+        mockSendFlowHistory,
+      );
+    });
+    it('doesnt append if current sendFlowHistory lengths doesnt match', async () => {
```

Comment:
Minor but line breaks between tests for readability?

Reply:
Sorry fixed on https://github.com/MetaMask/core/pull/1665/commits/39ac598f58cfc7746f2fd9b625c20d5788b5551e